src/content/guides/index.md

```diff
@@ -7,6 +7,6 @@ contributors:
 
 This section contains guides for understanding and mastering the wide variety of tools and features that webpack offers. The first is a simple guide that takes you through [installation](/guides/installation).
 
-The guides get more and more advanced as you go on. Most serve as a starting point, and once completed you should feel more comfortable diving into the actual [documentation](/configuration).
+The guides get more and more advanced as you go on. Most serve as a starting point and once completed you should feel more comfortable diving into the actual [documentation](/configuration).
```

Comment:
Thank you for the PR!
However, this change is really not required. The `and` is used in 2 kind of sentences.
1. Use a comma when the context is changed [I was playing football, and now I'm studying]
2. Don't use a comma when the context is not changed [for example, drinking and eating]

Reply:
Agreed, this is good without

Reply:
@hemal7735 There isn't any change of context, so comma must be removed. Also if we place a comma after `completed` it will make more sense.

Reply:
> @hemal7735 There isn't any change of context, so comma must be removed. Also if we place a comma after `completed` it will make more sense.

@EugeneHlushko please reply what you think about it.

Reply:
After reading this a few times i still feel like removing the comma here is not needed. Although, maybe adding a comma after `completed` will be a good addition

Reply:
I agree that removing the comma is not needed. The comma — along with the coordinating conjunction — separates two independent clauses, so it should be kept.